lambda_functions/process/make_vector_tiles/index.js

```diff
@@ -71,6 +73,44 @@ async function readGeojsonFiles(localDir) {
   return geojson;
 }
 
+
+async function emptyS3TilesDir(uuid, type) {
+  var S3 = new AWS.S3();
+  var keepRunning = true;
+  var params = {
+    Bucket: process.env.S3_BUCKET,
+    Prefix: `campaigns/${uuid}/render/${type}/tiles/`
+  };
+  var deletionsNumber = 0;
+
+  while (keepRunning) {
+    let listedObjects = await S3.listObjectsV2(params).promise();
+    let toDeleteItens = []
+
+    listedObjects.Contents.filter(
+      item => path.parse(item.Key).ext === '.pbf'
+    ).map(
+      item => toDeleteItens.push({ Key: item.Key })
+    );
```

Comment:
nitpick - I associate map/reduce with assignment. Since we aren't assigning the result of this operation to any variable, maybe replace `map` with `forEach`, so it's clear that we are simply running a list operation but not assigning the results to another variable. Or another way is to:

```
toDeleteItems = listedObjects.Contents.filter(
                  item => path.parse(item.Key).ext === '.pbf'
                ).map(item => { Key: item.key });
```